wallet/src/de/schildbach/wallet/ui/username/UsernameRequestsViewModel.kt

```diff
@@ -82,22 +95,12 @@ class UsernameRequestsViewModel @Inject constructor(
             .onEach { isShown -> _uiState.update { it.copy(showFirstTimeInfo = isShown != true) } }
             .launchIn(viewModelScope)
 
-        _filterState.flatMapLatest { filterState ->
+        _filterState.flatMapLatest {
             observeUsernames()
                 .map { duplicates ->
                     duplicates.groupBy { it.username }
                         .map { (username, list) ->
                             val sortedList = list.sortAndFilter()
-
-                            if (sortedList.isNotEmpty()) {
-                                val max = when (filterState.sortByOption) {
-                                    UsernameSortOption.VotesDescending -> sortedList.first().votes
-                                    UsernameSortOption.VotesAscending -> sortedList.last().votes
-                                    else -> sortedList.maxOf { it.votes }
-                                }
-                                sortedList.forEach { it.hasMaximumVotes = it.votes == max }
-                            }
```

Comment:
Not needed since the blue badge is for usernames you voted for, not the usernames with maximum votes.